Vagrantfile

```diff
@@ -6,72 +6,23 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
+cd /home/vagrant/oppia
+apt-get update && apt-get install -y python-pip python-dev unzip
+bash ./scripts/setup.sh
+bash ./scripts/install_third_party.sh
+SCRIPT
```

Comment:
My concern is running the `install_prerequisites.sh` with parameters, thus cascading changes into anything that might call it. What if anything else calls that script right now? 

A couple options: 

1. Have a timeout/break period in which people can cancel grabbing the packages.  

2. Have a check for the `VAGRANT` environment variable (already being exported) and run automatically if `VAGRANT` evaluates to `true`. 

3. Add a parameter that functions as the equivalent of `-y` in in `apt-get`. 

Option 2 is my preferred method.